controllers/articles-controllers.js

```diff
@@ -22,8 +22,23 @@ exports.getCommentsByArticleId = (req, res, next) =>{
   const promises=[findCommentsByArticleId(article_id), checkIdExists(article_id)]
   
     Promise.all(promises).then((resolvedPromises) =>{
+      
       const comments = resolvedPromises[0]
+      
       res.status(200).send(comments)
       
     }).catch(next)
+}
+
+exports.postComment = (req, res, next) => {
+  
+  const{article_id} = req.params
+  const comment = req.body
+  
+  const promises = [checkIdExists(article_id), insertComment(comment, article_id)]
+   
+  Promise.all(promises).then((resolvedPromises) =>{
+    
+    res.status(201).send(resolvedPromises[1])
```

Comment:
This should be sent on a key, for example, 'article'. This will make it easier to find when you're accessing this API later